fenix/app/src/main/java/org/mozilla/fenix/components/toolbar/RedesignToolbarFeature.kt

```diff
@@ -40,3 +43,37 @@ class IncompleteRedesignToolbarFeature(
     override val isEnabled: Boolean
         get() = settings.enableIncompleteToolbarRedesign
 }
+
+/**
+ * Returns the height of the bottom toolbar.
+ *
+ * The bottom toolbar can consist of a navigation bar,
+ * a combination of a navigation and address bar, or be absent.
+ */
+fun Context.getBottomToolbarHeight(): Int {
```

Comment:
💡 Think we can move this and `getTopToolbarHeight` to `Settings.kt` 

Also, thoughts on naming this to`bottomBarsHeight` since there can be two bars at the bottom and not just toolbar?

Reply:
I agree we could find a better place for the extension, but is `Settings.kt` a good fit? It doesn't have any extension functions.

It might be slightly confusing and lead readers to believe that there should be a different name for getting a single `bottomBarHeight`🤔 In context of toolbar translations, I believe it's better to think about toolbars as a top/bottom pair. From the `Behaviour` perspective, there are two `ScrollableToolbar` one at the top, one at the bottom, and sometimes they are there together. It doesn't really care about the contents, only about positioning. 
What do you think?

Reply:
`Settings` is injected with `appContext` so we have prefs and can access resources too. In the long run, each feature should have it's own prefs but since we use Settings.kt as the only pref service, think it makes sense to put it there. 

That's a good point and perspective to look at the toolbars and is in sync with the rest of the implementation. Top/bottom pair 👍